core/platform/datastore/gae_datastore_services.py

```diff
@@ -84,16 +86,29 @@ def update_timestamps_multi(entities, update_last_updated_time=True):
             update_last_updated_time=update_last_updated_time)
 
 
-def put_multi(models):
+def put_multi(Models):
```

Comment:
Why this change?

Reply:
The local lint checks failed after adding ```from core.platform import models```. It started warning me that ```models``` is in use. So, to circumvent that I changed ```models``` in ```put_multi``` to ```Models``` so it doesn't clash with the ```models```, and I kind of needed ```models``` to declare ```transaction_services = models.Registry.import_transaction_services()```

Reply:
I can revert the changes and go for something more subtle like:
```import core.platform.transactions.gae_transaction_services as transaction_servies```

Reply:
Oh, I see the issue now, instead of `Models` here use `entities`.

Reply:
On it